ginkgo.Jenkinsfile

```diff
@@ -175,7 +175,7 @@ pipeline {
                         TESTDIR="${GOPATH}/${PROJ_PATH}/test"
                     }
                     steps {
-                        sh 'cd ${TESTDIR}; ginkgo --focus=" Runtime*" -v --failFast=${FAILFAST} -- -cilium.provision=false'
+                        sh 'cd ${TESTDIR}; ginkgo --timeout=108m --focus=" Runtime*" -v --failFast=${FAILFAST} -- -cilium.provision=false'
```

Comment:
Please make these values configurable as environment variables instead of hardcoding.